src/assets/locales/ru.json

```diff
@@ -145,11 +145,15 @@
     "title": "Ошибка авторизации",
     "text": "Пожалуйста, внимательно проверьте корректность всех данных"
   },
+
   "googleAuthTostr": {
     "title": "Ошибка авторизации",
     "text": "Электронная почта не зарегистрирована в системе"
   },
-
+  "status": {
+   "change": "Сменить статус",
+   "save": "Сохранить статус"
+   },
```

Comment:
Bracket here not aligned.

Reply:
change is gives opportunity to change
save is PUT into backend